servicetalk-data-protobuf/build.gradle

```diff
@@ -31,14 +31,14 @@ dependencies {
   api project(":servicetalk-buffer-api")
   api project(":servicetalk-concurrent-api")
   api project(":servicetalk-serializer-api")
+  api "com.google.protobuf:protobuf-java:$protobufVersion"
```

Comment:
Consider using the [protobuf-bom](https://mvnrepository.com/artifact/com.google.protobuf/protobuf-bom)

Reply:
lets consider this as a followup PR and do it consistently throughout the project. `protobufVersion` is also used in plugins which I'm not sure can use the bom.

```gradle
protobuf {
  protoc {
    artifact = "com.google.protobuf:protoc:$protobufVersion"
  }
}
```

Reply:
Sure. I tried previously and it is still necessary to use `$protobufVersion` with the plugin.